pkg/controller/syncsetinstance/syncsetinstance_controller_test.go

```diff
@@ -80,6 +81,7 @@ func TestSyncSetReconcile(t *testing.T) {
 		expectDeleted          []deletedItemInfo
 		expectSSIDeleted       bool
 		expectErr              bool
+		expectApplied          *bool
```

Comment:
Why is this using a bool pointer? Why not always test the expected value of `applied` in every test case?

Reply:
Been awhile but I think I got confused around when we do or don't lookup the instance to do test.validate and this, non-pointer looks like it works fine, updated.

Reply:
Not pushed yet, but updated. :) Will wrap up the last one tomorrow.